### PR TITLE
give action performer environment variables

### PR DESCRIPTION
**Pull request by jeberl**
Summary
---------

This is needed to read actioner configs from the configs table

Test Plan
---------

Pushed to aws, ran with the following event, and saw that no errors emerged
```
{
  "Records": [
    {
      "body": "{\"ContentKey\": \"key\", \"ContentHash\": \"hash\", \"MatchingBankedSignals\": [{\"BankedContentId\": \"2862392437204724\", \"BankId\": \"bank 4\", \"BankSource\": \"te\", \"Classifications\": []}, {\"BankedContentId\": \"4194946153908639\", \"BankId\": \"bank 4\", \"BankSource\": \"te\", \"Classifications\": []}], \"ActionLabelValue\": \"EnqueForReview\"}"}
  ]
}
```

### CODE REVIEW

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (BarrettOlson) [inline hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:91]: Enqueue? (or are we saving chars?

File: hasher-matcher-actioner/terraform/actions/.terraform.lock.hcl
Comment (BarrettOlson) [inline hasher-matcher-actioner/terraform/actions/.terraform.lock.hcl:1]: what does this have a sub dir lock? it feels like this is an error but I'm not sure

**Review by BarrettOlson — APPROVED** [review]
Yay testability! (we should maybe make note of all the __main__ lambdas we want to refactor) 
Accepting to unblock 
Added a nit in line, please remove the extra lockfile unless you are sure it is needed.

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (BarrettOlson) [inline hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:60]: not json anymore I take it?

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (BarrettOlson) [inline hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:74]: should the bank names be different?

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (eriksendc) [inline hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:38]: collab_config_table --> config_table_name
HMA_CONFIG_TABLE --> CONFIG_TABLE_NAME

File: hasher-matcher-actioner/terraform/actions/main.tf
Comment (eriksendc) [inline hasher-matcher-actioner/terraform/actions/main.tf:101]: watch for my code coming in with this change but named a different way. here and further on.

File: hasher-matcher-actioner/terraform/actions/main.tf
Comment (eriksendc) [inline hasher-matcher-actioner/terraform/actions/main.tf:123]: This won't come in with my changes but you should use the naming from above.

File: hasher-matcher-actioner/terraform/actions/main.tf
Comment (eriksendc) [inline hasher-matcher-actioner/terraform/actions/main.tf:279]: The permission I used was scan, not get. Got that from the fetcher I though?

**Review by eriksendc — CHANGES_REQUESTED** [review]
I'd like to re-review after you rebase / merge changes in from master. Besides resolving conflicts, the only blocking issue is regarding the permissions you set up for your lambda to read from the config table.

File: hasher-matcher-actioner/terraform/actions/.terraform.lock.hcl
Comment (jeberl) [reply hasher-matcher-actioner/terraform/actions/.terraform.lock.hcl:1]: deleted, ran terraform init, and terraform apply. Everything seems fine

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (jeberl) [reply hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:60]: The `from_aws_message` includes json.loads

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (jeberl) [reply hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:74]: Never read? I think two matches in the same bank is possible

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (jeberl) [reply hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py]: Changed

File: hasher-matcher-actioner/terraform/actions/main.tf
Comment (jeberl) [reply hasher-matcher-actioner/terraform/actions/main.tf:284]: Hmm. Get* works for me. Any reason to change?

Comment (eriksendc) [reply hasher-matcher-actioner/terraform/actions/main.tf:284]: consistency?

**Review by eriksendc — APPROVED** [review]

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (BarrettOlson) [reply hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:74]: bank == dataset? or is bank a new concept?

**Pull request merged**
